datafusion/core/tests/sqllogictests/test_files/aggregate.slt

```diff
@@ -1390,7 +1390,7 @@ as values
  ('2021-1-1T05:11:10.432', 'Row 3');
 
 
-statement ok
+statement error DataFusion error: Arrow error: Parser error: Error parsing timestamp from '2021-1-1T05:11:10.432': error parsing date
```

Comment:
> 1. Change this test from

Will do

> 2 file an upstream ticket in arrow-rs to be more lenient

I would strongly request we do not do this, it will severely hurt performance for an incredibly esoteric use-case. It isn't actually documented that chrono supports this, and we have never intentionally supported it either